### PR TITLE
Fix (Memeooorr): Backend env configurations

### PR DESCRIPTION
**Pull request by OjusWiZard**
## Proposed changes
Add missing env configs for running memeooorr.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: frontend/constants/serviceTemplates.ts [hunk 1]
```diff
@@ -211,6 +211,37 @@ export const SERVICE_TEMPLATES: ServiceTemplate[] = [
...
+        name: 'DB path',
+        description: '',
+        value: 'logs/memeooorr.db',
+        provision_type: EnvProvisionType.FIXED,
+      },
+      TWIKIT_COOKIES_PATH: {
+        name: 'Twitter cookies path',
+        description: '',
+        value: 'twikit_cookies.json',
+        provision_type: EnvProvisionType.COMPUTED,
```
Comment (jmoreira-valory) [inline frontend/constants/serviceTemplates.ts:243]: does this require custom logic to fill?

Comment (OjusWiZard) [reply frontend/constants/serviceTemplates.ts:243]: unfortunately yes, it's added in the BE

**Review by jmoreira-valory — APPROVED** [review]

File: frontend/constants/serviceTemplates.ts (hunk 1 above)
Comment (Tanya-atatakai) [reply frontend/constants/serviceTemplates.ts:243]: we try to login on FE side in order to check if the provided creds are correct, maybe we can write this value? @mohandast52 wdyt

Comment (OjusWiZard) [reply frontend/constants/serviceTemplates.ts]: the relative path to this file will be `.operate/services/<service_config_id>/<TWIKIT_COOKIES_PATH>`

**Review by Tanya-atatakai — APPROVED** [review]

**Pull request merged**